MultiplayerMod/Multiplayer/State/MultiplayerGame.cs

```diff
@@ -4,14 +4,24 @@ namespace MultiplayerMod.Multiplayer.State;
 
 public static class MultiplayerGame {
 
+    public static IPlayer Player { get; set; } = null!;
```

Comment:
Hmmm, now we have two sources of truth.
It was supposed to use `IMultiplayerClient.Player` to get the current player.